### PR TITLE
Citizen Lab consent form

### PR DESCRIPTION
**Pull request by rpanah**
@ben-jones, please review (only the form contents were modified, but you should still check tags etc.).

### CODE REVIEW

**Comment by ben-jones** [comment]
Does this still satisfy IRB requirements? We can't change the form without IRB approval for a change to the protocol

**Comment by phillipa** [comment]
Stony Brook determined we are not human subjects research so we are switching to Toronto's form as we still have IRB approval there.

**Comment by ben-jones** [comment]
Cool. I will look over this in a bit more depth. The first thing that comes up after downloading and rendering is that people have to select which country they want to do informed consent for and submit the consent without changing their country. @rpanah, can you remind me if we autopopulate the country? If not, we should either autopopulate the country or don't show the informed consent submit button until they select a country so that we force users to see content relevant to their country before consenting.

**Comment by rpanah** [comment]
Is populated the same way as before.

File: static/informed_consent.html [hunk 1]
```diff
@@ -1,419 +1,413 @@
...
+          Electronic data will be stored in a secure server environment maintained at the University of
+          Toronto. Any personally identifiable data kept on computers outside of this secure server
+          environment will be encrypted at all times using strong encryption (128 bit AES) except to the
+          extent researchers need to decrypt them during use.
+          </p>
+          <p class="list-group-item-text">
+          Experiment data may include issuing requests for URLs found in the URL list (similar to what a
+          user does manually with a Web browser, such as Firefox, but automated), DNS queries (mapping
+          hostnames such as Google.com to IP address such as 72.14.52.3), or traceroutes (measuring
+          network paths). Experiments may log network traffic associated with the experiment (e.g.,network packets and timestamps). The software will also contain a unique user ID to allow us to
```
Comment (ben-jones) [inline static/informed_consent.html]: can we do a line wrap to 80 characters here please? The command is M-q in emacs

File: static/informed_consent.html [hunk 2]
```diff
@@ -1,419 +1,413 @@
...
+              <p>I confirm that I have the administrative authority to enroll my organization into this
+              study,</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I agree to take part in this study, and</p>
+            </div>
+
+            <div class="list-group-item">
+              <p>I will be given I can be provided with a copy of this signed and dated consent form.</p>
```
Comment (ben-jones) [inline static/informed_consent.html]: I think this is a typo. Was the typo part of the accepted protocol and can we change it?

File: static/initial_informed_consent.html [hunk 3]
```diff
@@ -1,182 +1,268 @@
...
+            <div class="list-group-item">
+              <p>
+              Freedom House "Freedom on the Net score” <br/>
+              http://www.freedomhouse.org/report/freedom-net/freedom-net-2012
+              </p>
+            </div>
+            <div class="list-group-item">
+              <p>
+              Economist "Democracy Index score"<br/>
+              http://www.eiu.com/public/thankyou_download.aspx?activity=download&campaignid=DemocracyIndex12
```
Comment (ben-jones) [inline static/initial_informed_consent.html]: What is the purpose of these links and can we remove them to ensure better privacy? The reason that we download all of the content to our server, then display what we downloaded is that we want to make sure that we aren't sending our users to the original source. If the user goes to the original source, then Freedom House or anyone else can have a list of our clients and we don't want that. Additionally, even if these are left in just for reference, there is still the potential that these links could identify the user due to browser prefetching (the browsing looking up the domains even though they are never visited)

Comment (rpanah) [inline static/initial_informed_consent.html]: While I agree with you, I'm not sure if we can remove them. I'll have to ask Citizen Lab folks about that.

File: static/informed_consent.html (hunk 2 above)
Comment (rpanah) [inline static/informed_consent.html]: It was part of it (I just copied text over). It should be fine to just correct it.

**Comment by rpanah** [comment]
@ben-jones, 
I have addressed all of the issues you mentioned (removed URLs from the form, fixed HTML formatting, corrected typo), and also fixed the bugs in all of your regular expressions (multiline wasn't enabled, so some instances didn't match) and enhanced them.
I also added a bit that removes the form tags (makes buttons and stuff go away, making it cleaner).
Please review again.

**Pull request merged**